src/org/exist/xquery/functions/map/AbstractMapType.java

```diff
@@ -1,11 +1,15 @@
 package org.exist.xquery.functions.map;
 
 import com.ibm.icu.text.Collator;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.exist.dom.QName;
 import org.exist.xquery.*;
 import org.exist.xquery.functions.fn.FunDistinctValues;
 import org.exist.xquery.value.*;
 
+import javax.xml.xpath.XPath;
+import java.util.AbstractMap;
```

Comment:
I can't see where XPath or AbstractMap may be used in this class?

Reply:
@wolfgangmm oops. Thanks. I have now removed those erroneous imports and rebased